js/mexc3.js

```diff
@@ -0,0 +1,3659 @@
+            //         "volume": "1732.461487",
+            //         "quoteVolume": null,
+            //         "openTime": 1641349500000,
+            //         "closeTime": 1641349582808,
+            //         "count": null
+            //     }
+            //
+            timestamp = this.safeInteger (ticker, 'closeTime');
+            bid = this.safeNumber (ticker, 'bidPrice');
+            ask = this.safeNumber (ticker, 'askPrice');
+            bidVolume = this.safeNumber (ticker, 'bidQty');
+            askVolume = this.safeNumber (ticker, 'askQty');
+            if (bidVolume === 0) {
+                bidVolume = undefined;
+            }
+            if (askVolume === 0) {
+                askVolume = undefined;
+            }
+            baseVolume = this.safeString (ticker, 'volume');
+            quoteVolume = this.safeString (ticker, 'quoteVolume');
+            open = this.safeString (ticker, 'openPrice');
+            high = this.safeNumber (ticker, 'highPrice');
+            low = this.safeNumber (ticker, 'lowPrice');
+            prevClose = this.safeString (ticker, 'prevClosePrice');
+            changeValue = this.safeString (ticker, 'priceChange');
+            changePcnt = this.safeString (ticker, 'priceChangePercent');
+            changePcnt = this.parseNumber (Precise.stringMul (changePcnt, '100'));
+        }
+        return this.safeTicker ({
+            'symbol': market['symbol'],
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'open': open,
+            'high': high,
+            'low': low,
+            'close': this.safeString (ticker, 'lastPrice'),
+            'bid': bid,
+            'bidVolume': bidVolume,
+            'ask': ask,
+            'askVolume': askVolume,
+            'vwap': undefined,
+            'previousClose': prevClose,
+            'change': changeValue,
+            'percentage': changePcnt,
+            'average': undefined,
+            'baseVolume': baseVolume,
+            'quoteVolume': quoteVolume,
+            'info': ticker,
+        }, market, false);
+    }
+
+    async fetchBidsAsks (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const [ marketType, query ] = this.handleMarketTypeAndParams ('fetchBidsAsks', undefined, params);
+        let method = undefined;
+        if (marketType === 'spot') {
+            method = 'spotPublicGetTickerBookTicker';
+            //
+            //     [
+            //       {
+            //         "symbol": "AEUSDT",
+            //         "bidPrice": "0.11001",
+            //         "bidQty": "115.59",
+            //         "askPrice": "0.11127",
+            //         "askQty": "215.48"
+            //       },
+            //     ]
+            //
+        } else if (marketType === 'swap') {
+            throw new BadRequest (this.id + ' fetchBidsAsks() is not available for ' + marketType + ' markets');
```

Comment:
I would argue that this is more "Not Supported" than a bad request

Reply:
right !